src/metadata.js

```diff
@@ -77,7 +77,7 @@ function evalClassification(label) {
     // Here for unknown reason, sometimes we find "Avis de dégrèvement" and sometimes "Avis dégrèvement"
     // We have to cover both possibilities
     label.match(
-      /Avis (de situation|de taxes?|d'impôt|de dégrèvement|dégrèvement|supplémentaire)/
+      /Avis (de situation|de taxes?|d'impôt|de dégrèvement|dégrèvement|supplémentaire|Montant de l'avance)/
```

Comment:
Are you sure about "Avis Montant de l'avance" ? It seems weird for a doc name

Reply:
Not first mistake
Second the ' is not the good one.

Now matching